source/_faq/release.markdown

```diff
@@ -1,12 +1,9 @@
 ---
 title: "Releases"
-description: "Releases"
+description: "Home Assistant release schedule"
 ha_category: Common
 ---
 
-The current release cycle is three weeks. This allows for two weeks of development time and 1 week of beta testing and fixing. Releases usually occur on a Wednesday. There is no fixed time of day when the release will happen because that depends on the person who is finishing the release, it could therefore be Tuesday or Thursday depending on your timezone when its available for you. If there was no announcement made in the previous release notes or on another communication channel then the release will happen.
+New major versions of Home Assistant are released every three weeks on a Wednesday. The exact dates can be seen in the upcoming events calendar on the [Home Assistant Developers](https://developers.home-assistant.io/) website. 
 
-Beta's are normally released on the 2nd Wednesday after the last major release. Please subscribe to the Beta Channel on the [Home Assistant Discord server](https://discord.gg/RMC8WkX) for help running beta versions.
-When a beta release is in progress, the release notes for it can be found under [Beta Release Notes](https://rc.home-assistant.io/latest-release-notes/).
-
-The release schedule is published in the upcoming events calender on the [Home Assistant Developers](https://developers.home-assistant.io/) website.
+This release schedule allows for two weeks of development time and one week of beta testing. Users who participate in the beta can view the changelog under [beta release notes](https://rc.home-assistant.io/latest-release-notes/) and get help in the `#beta` channel of [Home Assistant's Discord server](https://discord.gg/RMC8WkX). Testers are also encouraged to [report issues on GitHub](/help/#bugs-feature-requests-and-alike).
```

Comment:
This is not correct. We have a 3 week development cycle. With the release of the beta, the development of the next release starts as well.

Reply:
use `/join-chat` for linking to our Discord.

Reply:
```suggestion
The third week of our release schedule is primarily focused on beta testing. Users who participate in the beta can view the changelog under [beta release notes](https://rc.home-assistant.io/latest-release-notes/) and get help in the `#beta` channel of [Home Assistant's Discord server](/join-chat). Testers are also encouraged to [report issues on GitHub](/help/#bugs-feature-requests-and-alike).
```
Maybe this is better?